components/elements/buttons/ButtonLink/ButtonLink.test.ts

```diff
@@ -1,7 +1,15 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { setup, $fetch } from '@nuxt/test-utils-edge'
+
+import { describe, test as it } from 'vitest'
```

Comment:
Do we need `$fetch`, `describe` and `it` here? (As well as other test files that import these but don't seem to use em)

Reply:
when i was trying to get things working here, these were helping. So i'm going to leave them for now but its on the TODO to wrangle these tests and get them working in as concise a manner as possible :)

Reply:
Fair play. I'll create an issue for removing these then. Then you could add TODOs above these imports that link to the issue

Reply:
See https://github.com/leaderboardsgg/leaderboard-site/issues/503

Reply:
I'll add a TODO with a link in `vitest.config.ts` as I'll be touching that anyways to get vitest working